src/flp.rs

```diff
@@ -126,6 +131,9 @@ pub trait Type: Sized + Eq + Clone + Debug {
     /// The finite field used for this type.
     type Field: FftFriendlyFieldElement;
 
```

Comment:
Do you think the DP mechanism is specific to FLP? I would be inclined to think that it is more relevant to VDAF and should be applied at that level of abstraction. Simply put, I wonder if we need this API change here.